viz/charts/linkerd-viz/values.yaml

```diff
@@ -315,6 +315,12 @@ dashboard:
     # -- Pull policy for the  web component
     # @default -- defaultImagePullPolicy
     pullPolicy: ""
+  # -- dashboard service configuration
+  service:
+    # -- Additional annotations to add to dashboard service
+    annotations:
+      cloud.google.com/backend-config: '{"ports": {"8084":"linkerd-viz", "9994":"linkerd-viz"}}'
+      cloud.google.com/neg: '{"ingress": true}'
```

Comment:
I don't think we want to include these defaults for all deployments.

Reply:
Mea maxima culpa, I forgot to clean it up. Doing now.

Reply:
Done.